test/spec/PptTestData.scala

```diff
@@ -192,7 +193,8 @@ trait PptTestData extends RegistrationBuilder with MockAuthAction {
   protected val subscriptionStatus: SubscriptionStatus = SubscriptionStatus(
     subscriptionStatus = ETMPSubscriptionStatus.NO_FORM_BUNDLE_FOUND,
     idValue = "XXPPTP123456789",
-    idType = "ZPPT"
+    idType = "ZPPT",
```

Comment:
Worth removing the "Z" for consistency

Reply:
The idType in the repsonse, here, should stay as `ZPPT`, as per the spec. It's the regime that's corrected from `ZPPT` to `PPT`.

Reply:
I assumed the "Z" was a typo!  :-)